src/main/java/org/sonarlint/intellij/trigger/FileEditorTrigger.java

```diff
@@ -71,6 +74,8 @@ public void fileOpened(@NotNull FileEditorManager source, @NotNull VirtualFile f
     }
 
     analyzer.submitAsync(m, Collections.singleton(file), TriggerType.EDITOR_OPEN);
+
+    serverIssueUpdater.trackServerIssues(Collections.singleton(file));
```

Comment:
To synchronize both tasks at the end (only do server issue tracking after local analysis completes) we could share a CountDownLatch